genapi.py

```diff
@@ -908,12 +908,13 @@ def parse_args(argv):
 def setup(ns):
     """Ensure that we are ready to perform code generation. Returns typesystem."""
     # load raw table
-    dbtypes_json = os.path.join(ns.build_dir, 'dbtypes.json')
+    dbtypes_json = os.path.join(ns.src_dir, 'dbtypes.json')
     if not os.path.exists(ns.build_dir):
         os.mkdir(ns.build_dir)
     if not os.path.isfile(dbtypes_json):
-        print('Downloading ' + DBTYPES_JS_URL + ' ...')
-        f = urlopen(DBTYPES_JS_URL)
+        #print('Downloading ' + DBTYPES_JS_URL + ' ...')
```

Comment:
@gidden to double-check, is this acceptable to leave in the commented out code or should I delete it for the PR?